README.md

```diff
@@ -125,15 +125,6 @@ General guidelines:
    where `DATASET_GROUP_NAME` is the dataset nickname provided in the `yaml` file, e.g. `DYJetsToLL_M-50-madgraphMLM_ext1`
 
 
-## Running with ParticleNET
-
-In order to procude NanoAOD along with output branches from ParticleNET (which will be found in the "Jet" and "FatJet" collections for AK4 and AK8 outputs, respectively), the following additions are needed before executing the code:
-* Follow the installation recipe in the Readme [here](https://gitlab.cern.ch/rgerosa/particlenetstudiesrun2/-/tree/master/)
-* Then copy the PNET models into place where they're needed, according to the first section in the ReadMe [here](https://gitlab.cern.ch/rgerosa/particlenetstudiesrun2/-/tree/master/TrainingNtupleMakerAK4) and [here](https://gitlab.cern.ch/rgerosa/particlenetstudiesrun2/-/tree/master/TrainingNtupleMakerAK8).
-    * This will only work locally however. To make it work for jobs submitted to crab, do the following inside the CMSSW_BASE directory:
-      ```
-      git cms-addpkg RecoBTag/Combined
```

Comment:
if some packages/topics should be added/merged -> add them in installation steps in `env.sh`